README.md

```diff
@@ -277,6 +277,60 @@ fn main() {
 </nav>
 ```
 
+Positive and negative sections also apply to lists and maps. An empty list or map means a negative section and a non-empty one represents a positive section.
+
+
+#### Input
+
+```html
+
+{{+vacation}}
+<h1>Currenty on vacation</h1>
+<ul>
+{{/vacation}}
```

Comment:
I've pushed the current list to the data stack if a non-empty list is detected.

Try nesting the list inside the positive Boolean section and accessing the list using {{*.}}

I believe that might make the example easier to understand and simplify general usage.

Reply:
make sense, implemented